README.md

```diff
@@ -258,6 +273,7 @@ Arguments:
 - name: The name of the publication
 - collection: A function that returns the collection to publish (or just a collection, if it's insecure).
 - selector: (Optional) A function that returns mongo selector that will limit the results published (or just the selector).
+- settings: (Optional) A object with settings on server side's publish function. (Details see below)
```

Comment:
Well yes, cause than you could make it "switchable" in Form.
For the new Server-Side enhancement I didn't discover if some client side settings could be given to the publish function (since the server side already had a startup)